kolibri/utils/cli.py

```diff
@@ -579,12 +579,18 @@ def status():
 
 
 @main.command(cls=KolibriDjangoCommand, help="Start worker processes")
+@click.option(
+    "--port",
+    default=OPTIONS["Deployment"]["HTTP_PORT"],
+    type=int,
+    help="Port on which to run Kolibri services",
+)
 @click.option(
     "--background/--foreground",
     default=True,
     help="Run Kolibri services as a background task",
 )
-def services(background):
+def services(port, background):
```

Comment:
`kolibri-server` is not currently using the `services` command so this shouldn't cause any issues immediately. @jredrejo when we do switch `kolibri-server` to using the services command, would it be consistent with how everything is setup to explicitly tell the services command what port Kolibri is being served on?